### DISCUSSION TITLE
postgis DCIM folder is not located

### DISCUSSION THREAD

**Discussion by MonikaShah** [opengisch/QField#3268] [Q&A]
I was able to access postgis data to and fro to QFIELD Cloud. Postgis tables do show a link to captured image as DCIM/.....jpeg. 
But I was not able to locate any folder as DCIM on my postgis server. Neither those images are stored on the QFIELD CLoud. 
Any help is appreciated.

**Comment by suricactus** [comment]
I am not sure what is the relation between PostGIS and your DCIM directory. Please share your username/project combo, so we can try to guess what you are asking.

**Comment by MonikaShah** [comment]
Postgis layer has an attribute called photo whose widget type is attachment.
Wanted to know where these clicked photos are shared in postgis layer . The
data is stored like DCIM/... jpeg


username- jitendra
Organization - communitygis
Project- Prabhag70_KW



https://app.qfield.cloud/a/communitygis/Prabhag70_KW/

**Reply by pedromap** [reply]
I also have the same problem. If I work offline (export data from postgis to gpkg) i can store photos at the DCIM folder, case i work online, directaly to postgis, i can not store photos at DCIM, even if i creat the folder manually.

**Reply by suricactus** [reply]
Is this only cloud issue? Can you try with cable export of the project?

**Comment by MonikaShah** [comment]
Dear sir
This issue occurs when we have Remote postgis db on cloud.  Data sync
properly happens but the images clicked does not go to remote db location.
No folder named DCIM is created, neither the photos get stored if we
manually make DCIM folder there. In short, data sync is happening properly
but images are not getting transfered.

**Comment by suricactus** [comment]
I am still unable to reproduce. 

1) I have crated a single postgis point layer with the password saved within the `.qgz` file and action set to "Offline editing". 
2) This layer has single text field with widget "Attachment" and view "Image".
3) Uploaded the project to the cloud.
4) Downloaded the project to the device.
5) Created a new point with an attached image.
6) Pushed changes from QField.
7) The point appears on QFieldCloud and the file is accessible in the Files page too.

Please retry yourself or attach a super simple project that reproduces the issue.
